GoogleSignIn/Tests/Unit/GIDSignInInternalOptionsTest.m

```diff
@@ -29,6 +29,32 @@ @interface GIDSignInInternalOptionsTest : XCTestCase
 
 @implementation GIDSignInInternalOptionsTest
 
+- (void)testDefaultOptionsForVerificationFlow {
+#if TARGET_OS_IOS || TARGET_OS_MACCATALYST
+  id configuration = OCMStrictClassMock([GIDConfiguration class]);
+  id presentingViewController = OCMStrictClassMock([UIViewController class]);
+  NSString *loginHint = @"login_hint";
+  
+  GIDVerifyCompletion completion = ^(GIDVerifiedAccountDetailResult *_Nullable verifiedResult,
+                                     NSError * _Nullable error) {};
+  GIDSignInInternalOptions *options =
+  [GIDSignInInternalOptions defaultOptionsWithConfiguration:configuration
+                                   presentingViewController:presentingViewController
+                                                  loginHint:loginHint
+                                              addScopesFlow:NO
+                                           verifyCompletion:completion];
+  
+  XCTAssertTrue(options.interactive);
+  XCTAssertFalse(options.continuation);
+  XCTAssertFalse(options.addScopesFlow);
+  XCTAssertNil(options.extraParams);
+  XCTAssertEqual(options.accountDetailsToVerify, @[]);
```

Comment:
Are you able to test passing in the account detail?

Reply:
Yep, added!